frontend/components/Fridge/FridgeIngredients.js

```diff
@@ -134,6 +134,7 @@ function FridgeIngredients() {
       <FridgeIngredientsForm
         onSubmit={handleSubmit}
         ingredientToEdit={ingredientToEdit}
+        setIngredientToEdit={setIngredientToEdit}
```

Comment:
I think it would be cleaner to pass a prop named `resetIngredientToEdit` that is just callable without argument to the form component, to avoid giving it more power than needed.

Reply:
Ok I will do that :+1: